src/python/pants/core/goals/lint.py

```diff
@@ -175,7 +175,10 @@ def tool_name(cls) -> str:
 
     @classproperty
     def tool_id(cls) -> str:
-        """The "id" of the tool, used in tool selection (E.g. --only=<id>)."""
+        """The "id" of the tool, used in tool selection (E.g.
+
+        --only=<id>).
```

Comment:
Bad change

Reply:
How so?

Reply:
Why is it split on two lines, with an empty line in between? This is one sentence.

If the line was too long to fit, then remove the middle blank line.

Reply:
Oh thats what you meant

Reply:
I think docformatter is being very dumb here...